test/JDBC/expected/db_securityadmin-vu-verify.out

```diff
@@ -157,13 +159,49 @@ GO
 EXEC babel_5135_roleop_proc1;
 GO
 
--- CASE 3.2 - ADD/DROP the membership of user-defined database roles
+-- CASE 3.2.a - ADD/DROP the membership of user-defined database roles
 ALTER ROLE babel_5135_r1 ADD MEMBER babel_5135_u1;
 GO
 
 ALTER ROLE babel_5135_r1 DROP MEMBER babel_5135_u1;
 GO
 
+-- CASE 3.2.b - ADD/DROP the membership of user-defined database roles to/from current_user
+SELECT current_user;
+GO
+~~START~~
+varchar
+babel_5135_dbsecadmin_u1
+~~END~~
+
+
+ALTER ROLE babel_5135_r1 ADD MEMBER babel_5135_dbsecadmin_u1;
+GO
+
+ALTER ROLE babel_5135_r1 DROP MEMBER babel_5135_dbsecadmin_u1;
+GO
+
+-- CASE 3.2.c - It should be able to ALTER the membership of database role which is member of db_owner
+ALTER ROLE db_owner ADD MEMBER babel_5135_r1;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Adding database roles to db_owner is not currently supported in Babelfish)~~
```

Comment:
The comment above says it should be able ALTER the membership but here we are getting an error. Is this error expected? If yes then we should chose a different test case in which current user is able to ALTER the membership of db_owner.

Reply:
This is added for future when we add support of adding database roles as member of db_owner.